.github/workflows/ci.yml

```diff
@@ -370,7 +370,109 @@ jobs:
         sudo apt-get install -y tesseract-ocr-kor
         sudo apt-get install diffstat
         tesseract --version
-        ./test_unstructured_ingest/test-ingest.sh
+        ./test_unstructured_ingest/test-ingest-src.sh
+
+
+  test_ingest_dest:
+    environment: ci
+    strategy:
+      matrix:
+        python-version: ["3.8","3.9","3.10","3.11"]
```

Comment:
maybe we only run on 3.10 until we have a reason to have a matrix dest job? ..this will save some ci time for those other versions spinning up only to skip all destination tests

Reply:
actually, I guess maybe there are some we do want to run against. hmm, one sec

Reply:
Since there are only so many of these, I have them all being run for every version right now.

Reply:
let's run: azure, gcs, and s3 on all version. let's only run the others on 3.10